misk-clustering-dynamodb/src/test/kotlin/misk/clustering/DynamoClusterTest.kt

```diff
@@ -0,0 +1,62 @@
+package misk.clustering
+
+import jakarta.inject.Inject
+import misk.MiskTestingServiceModule
+import misk.aws2.dynamodb.testing.DynamoDbTable
+import misk.aws2.dynamodb.testing.InProcessDynamoDbModule
+import misk.clustering.dynamo.DyClusterMember
+import misk.clustering.dynamo.DynamoClusterModule
+import misk.clustering.dynamo.DynamoClusterWatcherTask
+import misk.clustering.weights.FakeClusterWeightModule
+import misk.inject.KAbstractModule
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import misk.time.FakeClock
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.time.Duration
+
+private class TestModule : KAbstractModule() {
+  override fun configure() {
+    install(MiskTestingServiceModule())
+    install(FakeClusterWeightModule())
+    install(
+      InProcessDynamoDbModule(
+        DynamoDbTable("misk-cluster-members", DyClusterMember::class),
+      )
+    )
+    install(DynamoClusterModule())
+  }
+}
+
+@MiskTest(startService = true)
+class DynamoClusterTest {
+  @MiskTestModule
+  private val module = TestModule()
+
+  @Inject private lateinit var cluster: Cluster
+  @Inject private lateinit var dynamoClusterWatcherTask: DynamoClusterWatcherTask
+  @Inject private lateinit var fakeClock: FakeClock
+
+  @Test
+  fun basic() {
```

Comment:
going to add the following tests, but wanted to get earlier review:
- more than 100 entries in table (tests pagination)
- more than 1 entry in table (just sanity)
- inactive cluster doesn't put in an entry
- resource mapper still works